spec/spec_helper.rb

```diff
@@ -16,6 +16,8 @@
 
 Capybara.run_server = false
 
+Capybara.enable_aria_label = true
```

Comment:
Allows for matching aria labels! Which we need to find the edit collection link in the H2 spec.